### PR TITLE
Rewrite README

### PR DESCRIPTION
**Pull request by jvanhoefer**
The RADME got reworked/shortened. I guess, it has everything important in it, all new features are in there and all links should work again ;)

### CODE REVIEW

**Review by martamatos — COMMENTED** [review]
I have a few suggestions here:
 - updating the README only when:
   - the package is on PyPi and get the `pip install <package>` straight away
   - the documentation is on readthedocs, so we can write in the `About` something like "For more details check the documentation [here](link)"
 - I would leave the `Usage` section, maybe shorter, but with all the commands, especially the CLI ones. Personally i like to check the README and be able to give a quick try to the package without going into the full documentation. 
 
 What do you think?
 
 And thanks for updating the links to the badges, was thinking of that too :)

File: README.md [hunk 1]
```diff
@@ -1,123 +1,53 @@
 # yaml2sbml
 
 [![Codacy Badge](https://app.codacy.com/project/badge/Grade/dc25c9a84ba54710bbb23a6e08ab5d22)](https://www.codacy.com/manual/martamatos/yaml2sbml/dashboard?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=yaml2sbml-dev/yaml2sbml&amp;utm_campaign=Badge_Grade)
```
Comment (martamatos) [inline README.md:3]: i think this link is also wrong xD

**Comment by jvanhoefer** [comment]
> I have a few suggestions here:
> 
> * updating the README only when:
>   
>   * the package is on PyPi and get the `pip install <package>` straight away
>   * the documentation is on readthedocs, so we can write in the `About` something like "For more details check the documentation [here](link)"
> * I would leave the `Usage` section, maybe shorter, but with all the commands, especially the CLI ones. Personally i like to check the README and be able to give a quick try to the package without going into the full documentation.
> 
> What do you think?

Sounds good, I would like to go to a "master/develop" branch organization of our development, and also would like to _always_ have a new version, if things in the master branch change (we were a bit messy there until now...). So if you are fine with that, I would set up this stuff and upload it the current version to pypi (with version 0.2.0 as the first pypi version)

**Comment by jvanhoefer** [comment]
Since the tool is on pypi now, I finalized the README now :) From my side ready for merging.

File: README.md [hunk 2]
```diff
@@ -1,123 +1,81 @@
...
+# SBML conversion
+yaml2sbml.yaml2sbml(input_yaml_dir, output_sbml_dir)
 
-Here all inputs ar given as strings.
-
-To generate PEtab files call `yaml2petab` via
+#PEtab conversion
+yaml2sbml.yaml2petab(input_yaml_dir, 
+                     output_petab_dir,
+                     sbml_name)
```
Comment (dilpath) [inline README.md:63]: The argument names here differ from the definitions, e.g. from `yaml2sbml.py` and `yaml2PEtab.py`:
```python
def yaml2sbml(yaml_file: str, sbml_file: str):
def yaml2petab(yaml_file: str,
               output_dir: str,
               model_name: str,
               petab_yaml_name: str = None,
               measurement_table_name: str = None):
```

Could be changed for consistency.

File: README.md [hunk 3]
```diff
@@ -1,123 +1,81 @@
...
+And in the command line via 
+```shell
+# SBML conversion
+yaml2sbml <yaml_input_file> <sbml_output_file>
 
-yaml2sbml.yaml2petab(yaml_file,
-                     output_dir,
-                     model_name)
+#PEtab conversion
+yaml2petab <yaml_input_file> <petab_output_directory> <model_name>
```
Comment (dilpath) [inline README.md:73]: Similar to the above comment, the argument names here could also be changed to match the definitions (if this is how the arguments are named in e.g. `yaml2sbml -h` then also fine)

**Review by dilpath — APPROVED** [review]
:+1:

File: README.md [hunk 4]
```diff
@@ -1,123 +1,81 @@
...
 
-* [format_documentation.md](format_documentation.md) provides a documentation of the format of the input yaml. 
+`yaml2sbml` is a small package in Python to convert an ODE model specified in a YAML file into an 
+[**SBML**](http://www.sbml.org/) for ODE simulation and into 
+[**PEtab**](https://github.com/martamatos/yaml2sbml) for parameter fitting. `yaml2sbml` offers:
 
-* Examples can be found in the `examples` folder.
+* Translate ODE models specified in YAML into SBML/PEtab via a Python and a command line interface.
+* A format validator for the input YAML.
+* A model editor, that allows to generate, import and export a YAML models.
```
Comment (yannikschaelte) [inline README.md:28]:
```suggestion
* A model editor that allows to generate, import and export YAML models.
```

File: README.md [hunk 5]
```diff
@@ -1,123 +1,81 @@
...
-#### yaml2sbml command
-
-To convert an ODE model encoded in a yaml file to SBML using the terminal, go to the `yaml2sbml` folder and run:
+`yaml2sbml`can be `pip`-installed via
 
 ```shell
- yaml2sbml <yaml_input_file> <sbml_output_file>
+pip install yaml2sbml
 ```
+For more infos see the [docs](https://yaml2sbml.readthedocs.io/en/latest/#).
```
[the remark this answers is not in the document]
Comment (yannikschaelte) [reply README.md:37]: maybe link directly to installation? (/install.html)

File: README.md [hunk 6]
```diff
@@ -1,123 +1,81 @@
...
-                     model_name)
+#PEtab conversion
+yaml2petab <yaml_input_file> <petab_output_directory> <model_name>
 ```
-Here `yaml_file, output_dir` and `model_name` are strings.
 
-## Known issues and limitations
+### Format Validation
+
+Format validation is possible in Python via `yaml2sbml.validate_yaml` and in command line via `yaml2sbml_validate`.
```
Comment (yannikschaelte) [inline README.md:78]:
```suggestion
Format validation is possible in Python via `yaml2sbml.validate_yaml` and in the command line via `yaml2sbml_validate`.
```

**Review by yannikschaelte — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (jvanhoefer) [reply README.md:73]: They should be :+1: :)

File: README.md (hunk 2 above)
Comment (jvanhoefer) [reply README.md:63]: You are right, but I guess, that the names in `yaml2petab.py` are not as descriptive as possible. Are you. in favor of changing the names `yaml2petab.py` then?

Comment (dilpath) [reply README.md:63]: I think it's OK. But in general, as there are "files", "filenames" and "directories", it doesn't make much sense to name the input `input_yaml_dir` (as in the README), when it's not a directory, or `yaml_file` (as in the method definition), when it's not a file object like
```python
with open('my_yaml.yml', 'r') as yaml_file:
    yaml2petab(yaml_file, ...)
```
If the method was overloaded to accept file objects as well, like the above example, then `yaml_file` makes more sense as a name for type `Union[str, TextIO]`. If kept as-is, I would opt for `yaml_filename` (but really, fine as-is in the definition).

Similarly, `model_name` is less obvious than `sbml_filename` to me (I think `sbml_filename` reflects how it is used?).

I think simply changing the usage in this README to match the method definition would be an improvement.

Comment (jvanhoefer) [reply README.md:63]: You are right, probably the naming here is a bit sloppy, I would therefore rather go for renaming the names in the function definition... :)

**Pull request merged**